resources/views/users/show.blade.php

```diff
@@ -38,12 +40,11 @@
                             <th class="table__cell">Items</th>
                             <th class="table__cell">Quantity</th>
                             <th class="table__cell">Updated At</th>
-                            <th class="table__cell">Flagged</th>
```

Comment:
I didn't think so. Especially since what we actually care about is the statuses of the RB items, not the Reportback parent object.